tools/ghidra_scripts/export_to_redux.py

```diff
@@ -0,0 +1,32 @@
+import os
```

Comment:
Could we add an entry in the README of this folder to quickly describe the function of the script and how to use it?